Dockerfile

```diff
@@ -15,11 +15,13 @@ RUN INSTALL_PKGS=" \
 COPY --from=builder /go/src/github.com/openshift/cluster-logging-operator/_output/bin/cluster-logging-operator /usr/bin/
 COPY scripts/* /usr/bin/scripts/
 COPY files/ /usr/bin/files/
+ADD controller-manifests /manifests
 # this is required because the operator invokes a script as `bash scripts/cert_generation.sh`
 WORKDIR /usr/bin
 ENTRYPOINT ["/usr/bin/cluster-logging-operator"]
 LABEL io.k8s.display-name="OpenShift cluster-logging-operator" \
       io.k8s.description="This is a component of OpenShift Container Platform that manages the lifecycle of the Aggregated logging stack." \
       io.openshift.tags="openshift,logging,cluster-logging" \
       io.openshift.release.operator=true \
```

Comment:
@jwforres i seem to recall there was an open question about whether these images were getting pulled into the CVO payload because of this..did we get to the bottom of that?  Should this be removed?

Reply:
@jcantrill it doesn't have to block the PR but you're going to want to run this down one way or another.